dashboard/app/assets/javascripts/client_state.js

```diff
@@ -136,5 +136,29 @@ dashboard.clientState.recordVideoSeen = function (videoId) {
   }
 };
 
+dashboard.clientState.hasSeenCallout = function(calloutId) {
+  var calloutsSeenJson = localStorage.getItem('calloutsSeen') || '{}';
+  try {
+    var calloutsSeen = JSON.parse(calloutsSeenJson);
+    return calloutsSeen[calloutId] == 1;
```

Comment:
Changed to 'true' - there's no real value in being frugal here given how much storage there is